### DISCUSSION TITLE
欢迎来到 Wot Design Uni 讨论区！

### DISCUSSION THREAD

**Discussion by Moonofweisheng** [Moonofweisheng/wot-design-uni#14] [Show and tell]
## <img src="https://camo.githubusercontent.com/8653492b3ab0c46cc580ad293f0555880ecf8ac82f0a761f17af1335e85e4de6/68747470733a2f2f71706c7573706963747572652e6f73732d636e2d6265696a696e672e616c6979756e63732e636f6d2f364c6a6a51412f48692e676966" height="20"> 欢迎  ！

我们正在使用“Discussion”作为与社区其他成员联系的地方。 在这里你可以：

- 提出你在使用 Wot Design Uni 过程中遇到的问题。
- 分享你对 Wot Design Uni 的反馈和建议。
- 与社区的其他成员互动。

**Comment by LP-J** [comment]
占位符

**Comment by missyouliu** [comment]
```日历组件实现滑动切换```
